ocrd/ocrd/processor/base.py

```diff
@@ -354,6 +354,9 @@ def zip_input_files(self, require_first=True, mimetype=None, on_error='skip'):
             # might be because of invalid page_id (range)
             if self.page_id and not files_:
                 LOG.warning(f"Could not find any files for --page-id {self.page_id} - compare '{self.page_id}' with the output of 'orcd workspace list-page'.")
+                if on_error == 'abort':
+                    raise ValueError(f"Could not find any files for --page-id {self.page_id} - compare "
+                                     f"'{self.page_id}' with the output of 'orcd workspace list-page'.")
```

Comment:
```suggestion
                msg = f"Could not find any files for --page-id {self.page_id} - compare '{self.page_id}' with the output of 'orcd workspace list-page'."
                raise ValueError(msg) if on_error == 'abort' else LOG.warning(msg)
                    
```

Reply:
```python
raise ValueError(msg) if on_error == 'abort' else LOG.warning(msg)
```
is a wrong syntax since Python tries to `raise LOG.warning(msg)` when the flag is not `abort`.

Reply:
Ah, yeah, sry, not tested, was just a proposal.

Reply:
> ```python
> (raise ValueError(msg)) if on_error == 'abort' else LOG.warning(msg)
> ```

might work. but better not to be too clever for my own good. I just meant that the message could be a variable to be used for both warning log and raised exception.